<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ticket_Reservation</class>
 <widget class="QDialog" name="Ticket_Reservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1021</width>
     <height>469</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Tickets</string>
    </attribute>
    <widget class="QTabWidget" name="WidgetR">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>381</width>
       <height>271</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="AjoutR">
      <attribute name="title">
       <string>Ajout et Modification</string>
      </attribute>
      <widget class="QPushButton" name="AjoutTicket">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ref_input">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom du film</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference Ticket</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prix_input">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix Ticket</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_input">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="EditTicket">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="RechercheR">
      <attribute name="title">
       <string>Recherche</string>
      </attribute>
      <widget class="QLineEdit" name="Idf_find">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="TriR_Idf_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cCritere">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>60</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>REFERENCE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PRIX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NOM</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="SuppressionR">
      <attribute name="title">
       <string>Suppression</string>
      </attribute>
      <widget class="QPushButton" name="RemoveTicket">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>70</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference Ticket</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ref_remove">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QTableView" name="TabR1">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>517</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>141</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Tri </string>
     </property>
     <widget class="QPushButton" name="TriR_Idr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference Ticket</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TriR_Idf">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix Ticket</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TriR_Idc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom Film</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Imprimer">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>372</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PDF">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_10">
    <attribute name="title">
     <string>Reservations</string>
    </attribute>
    <widget class="QTabWidget" name="WidgetF">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>381</width>
       <height>271</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="AjoutF">
      <attribute name="title">
       <string>Ajout et Modification</string>
      </attribute>
      <widget class="QPushButton" name="AjoutReservation">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nombre_input">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference res</string>
       </property>
      </widget>
      <widget class="QPushButton" name="EditF_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="date_input">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="refr_input">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="RechercheF">
      <attribute name="title">
       <string>Recherche</string>
      </attribute>
      <widget class="QLineEdit" name="rechercheInput">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>fefef</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="bRechReservation">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>100</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cCritereRes">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>60</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>REFERENCE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NOMBRE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATES</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="SuppressionF">
      <attribute name="title">
       <string>Suppression</string>
      </attribute>
      <widget class="QPushButton" name="RemoveReservation">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Idf_remove">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>70</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Reference Reservation</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="ResetF_button">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>410</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>111</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Tri par</string>
     </property>
     <widget class="QPushButton" name="TriF_Id">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference RES</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TriF_Nom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TriF_Prenom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="TabF1">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>517</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="PDFRes">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>308</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="impRes">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
